Doc/howto/logging-cookbook.rst

```diff
@@ -1634,7 +1634,7 @@ method. Since then, Python has gained two new formatting approaches:
 
 Logging (as of 3.2) provides improved support for these two additional
 formatting styles. The :class:`Formatter` class been enhanced to take an
-additional, optional keyword parameter named ``style``. This defaults to
+additional, optional keyword argument named ``style``. This defaults to
```

Comment:
I think an alternative change, not necessarily better, would have been /take/have/.